libs/components/pages/src/lib/modules/action-hub/action-hub-buttons.component.scss

```diff
@@ -1,7 +1,5 @@
 :host {
   display: block;
-
-  &:not(:empty) {
-    margin-bottom: var(--sky-margin-stacked-xl);
-  }
+  position: relative;
+  top: calc(-1 * var(--sky-margin-stacked-sm));
```

Comment:
Can you provide some context to this change? Using `position: relative` is a bit risky since it will ignore the margin of its parent.

Reply:
I've removed this piece. There is a visual shift, but it seems like this spacing better matches the [action hub design](https://developer.blackbaud.com/skyux/components/action-hub?docs-active-tab=design).